iac-extensions/docker/src/main/java/org/sonar/iac/docker/parser/grammar/DockerGrammar.java

```diff
@@ -146,7 +149,7 @@ public ExposeTree EXPOSE() {
   public PortTree PORT() {
     return b.<PortTree>nonterminal(DockerLexicalGrammar.PORT).is(
       b.firstOf(
-        f.port(b.token(DockerLexicalGrammar.NUMERIC_LITERAL), b.token(DockerLexicalGrammar.SEPARATOR_PORT), b.optional(b.token(DockerLexicalGrammar.STRING_LITERAL_WITHOUT_SPACE))),
+        f.port(b.token(DockerLexicalGrammar.NUMERIC_LITERAL), b.token(DockerLexicalGrammar.SEPARATOR_PORT), b.optional(b.token(STRING_LITERAL_WITHOUT_SPACE))),
```

Comment:
We have to find a convention when to use static imports. I wouldn't do it here. Please add lien breaks.